packages/grz-db/src/grz_db/models/submission.py

```diff
@@ -281,18 +317,21 @@ def add_submission(
                 raise
 
     def modify_submission(self, submission_id: str, key: str, value: str) -> Submission:
-        if key.casefold() in {"tan-g", "tan_g", "tang"}:
+        key = key.casefold()
+        if key in {"tan-g", "tan_g", "tang"}:
             key = "tan_g"
-        elif key.casefold() in {"pseudonym", "donor-pseudonym", "donor_pseudonym"}:
+        elif key in {"pseudonym", "donor-pseudonym", "donor_pseudonym"}:
             key = "pseudonym"
```

Comment:
I think we can remove these special cases again, I initially just put them in for convenience.

Reply:
I agree we should remove them, but we need some way to show the internal column/key names to the user, because currently they would have to know or guess the names, since `grzctl db submission show` shows nicer key names. e.g. `tan_g` to `tanG`.

Reply:
I.e., list the field names from the SubmissionBase basically (snake_case / camelCase)?